data/crac-file/crac-io-json/src/test/resources/file.json

```diff
@@ -0,0 +1,424 @@
+{
+  "@c" : ".SimpleCrac",
+  "id" : "idCrac",
+  "name" : "name",
+  "cnecs" : [ {
+    "@c" : "com.farao_community.farao.data.crac_impl.SimpleCnec",
+    "id" : "idCnec",
+    "name" : "Cnec",
+    "criticalNetworkElement" : {
+      "@c" : ".NetworkElement",
+      "id" : "idMR",
+      "name" : "Monitored Element"
+    },
+    "threshold" : {
+      "@c" : ".AbsoluteFlowThreshold",
+      "unit" : "AMPERE",
+      "side" : "RIGHT",
+      "direction" : "OUT",
+      "maxValue" : 999.0
+    },
+    "state" : {
+      "@c" : "com.farao_community.farao.data.crac_impl.SimpleState",
+      "contingency" : {
+        "@c" : "com.farao_community.farao.data.crac_impl.ComplexContingency",
+        "id" : "idContingency",
+        "name" : "My contingency",
+        "networkElements" : [ {
+          "@c" : ".NetworkElement",
+          "id" : "idLine2",
+          "name" : "My Line 2"
+        }, {
+          "@c" : ".NetworkElement",
+          "id" : "idLine3",
+          "name" : "My Line 3"
+        }, {
+          "@c" : ".NetworkElement",
+          "id" : "idNE1",
+          "name" : "My Element 1"
+        } ]
+      },
+      "instant" : {
+        "@c" : ".Instant",
+        "duration" : 200.0
+      }
+    }
+  }, {
+    "@c" : "com.farao_community.farao.data.crac_impl.SimpleCnec",
+    "id" : "idCnec2",
+    "name" : "Cnec 2",
+    "criticalNetworkElement" : {
+      "@c" : ".NetworkElement",
```

Comment:
Why do we need to precise that this is a NetworkElement, can it be something else ?

Reply:
no